content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,108 @@ aliases:
 ---
 
 Use the `influxd upgrade` command to upgrade InfluxDB 1.x to InfluxDB 2.0.
-This command copies all data stored in 1.x [databases](/influxdb/v1.8/concepts/glossary/#database) and
+The `upgrade` command copies all data stored in 1.x [databases](/influxdb/v1.8/concepts/glossary/#database) and
 [retention policies](/influxdb/v1.8/concepts/glossary/#retention-policy-rp)
 to 2.0 [buckets](/influxdb/v2.0/reference/glossary/#bucket).
 
+<!-- TODO summarize effects on other resources -->
+
+## Before you begin: important considerations
+
+Before upgrading to InfluxDB 2.0, please consider the following issues;
+some or all might apply to your specific installation and use case.
+The sections below contain our recommendations for addressing possible gaps in the upgrade process.
+You may also find while reading that you'd like to hold off on upgrading for now.
+
+### Continuous queries
+
+When upgrading from InfluxDB 1.x and using continuous queries (CQs) to downsample data,
+we recommend extracting CQs from your InfluxDB 1.x instance prior to upgrade with `show continuous queries`.
+Run the following:
+
+```sh
+$ influx
+Connected to http://localhost:8086 version 1.8.3
+InfluxDB shell version: 1.8.3
+> show continuous queries
+```
+
+Save this output so it can be used later.
+
+Continuous Queries are replaced by Tasks in InfluxDB 2.0
+To convert your CQs to Flux, see the [Tasks documentation](/influxdb/v2.0/process-data/get-started/).
+
+### Supported protocols
+
+If you're using one of the [supported protocols](/influxdb/v1.8/supported_protocols/) in InfluxDB 1.x (CollectD, Graphite, OpenTSDB, Prometheus, UDP),
+you may want to research using Telegraf instead.
+InfluxDB 2.0 doesn't support these protocols directly.
+Use Telegraf to translate from the source protocol to InfluxDB 2.0.
+
+### Kapacitor
+
+To continue using Kapacitor, update the Kapacitor configuration to use the appropriate security credentials per [1.x compatibility API docs 1]()
+(using the appropriate user and token to allow access).
+
+### Dashboards
+
+There is no [subscriptions API](/influxdb/v1.8/administration/subscription-management/) in InfluxDB 2.0.
+To continue to use existing dashboarding tools such as Chronograf and Grafana with InfluxDB 2.0,
+reconfigure your data sources to point to your new InfluxDB 2.0 OSS instance with the appropriate security credentials.
+
+### User migrations
+
+`influxd upgrade` attempts to migrate existing 1.x users and their permissions.
+However, *administrative users are not copied*.
+
+If using an admin user for vizualition or Chronograf administrative functions, you may want to create a new read only user first.
+Admin capabilities are to the primary user created in the 2.0 setup process via the `influxd upgrade` command.
```

Comment:
Admin rights are granted to the primary user created in the 2.0.....

This provides you with the opportunity to re-assess who should be granted admin level access as a part of your InfluxDB 2.0 setup.